bootstrap/glide.yaml

```diff
@@ -36,7 +36,7 @@ import:
 - package: k8s.io/apiserver
   version: kubernetes-1.10.2
 - package: github.com/ksonnet/ksonnet
-  version: ~0.11.0
+  version: ~0.12.0
```

Comment:
Sorry forgot to mention earlier:
glide.lock need to be updated as well. 
We can run ```glide up``` so that glide.lock pick up changes in glide.yaml
(glide install will use glide.lock to determine versions)